fhirclient/client.py

```diff
@@ -7,7 +7,7 @@
 
 from .server import FHIRServer, FHIRUnauthorizedException, FHIRNotFoundException
 
-__version__ = '4.2.0'
+__version__ = '4.3.0'
```

Comment:
This fix is only worth a patch version bump, but since `main` includes some new pagination code, I'm bumping the minor version too.

But let's chat in person about next steps - I might lean towards a small 4.2.x branch for just this patch since more pagination changes are in-flight.